docs/servers.md

```diff
@@ -10,12 +10,14 @@ You can connect to one using the "Join Game" button under the "Play" menu. Unlik
 
 To establish a dedicated server, a dedicated Linux or Windows machine is **highly** recommended.
 
-1. If you haven't already, install at least JRE and JDK 8.
```

Comment:
JDK is only necessary if you want to build the server jar from source. If you know how to do that, you don't probably don't need to be told to install it.